src/gui/mzroll/mzroll.pro

```diff
@@ -48,11 +47,13 @@ win32 {
 }
 
 mac {
-    INCLUDEPATH  += $$top_srcdir/3rdparty/google-breakpad/src/
-    QMAKE_LFLAGS += -L$$top_builddir/libs/
+    QMAKE_CXXFLAGS += -fopenmp
```

Comment:
Added this flag already on line 15

Reply:
breakpad and pthread is also common to all 3 platforms. Can't that be moved out of the individual blocks?

Reply:
> Added this flag already on line 15

This has been removed  in the next set of commits. 

> breakpad and pthread is also common to all 3 platforms

Breakpad is but  I doubt pthread is. Not really sure why pthread is added in the Mac section

> Can't that be moved out of the individual blocks?

It can be. I plan to fix/refactor our pro files in the coming time, do you want me to fix this in this PR?

Reply:
Not exactly related to this PR so you can do it later if you want. (unless it's the bucket list kind of 'later' :P )